src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -223,7 +225,7 @@ public enum ProtocolType
     public sealed partial class SafeSocketHandle : Microsoft.Win32.SafeHandles.SafeHandleMinusOneIsInvalid
     {
         public SafeSocketHandle() : base (default(bool)) { }
-        public SafeSocketHandle(System.IntPtr preexistingHandle, bool ownsHandle) : base (default(bool)) { }
+        public SafeSocketHandle(nint preexistingHandle, bool ownsHandle) : base (default(bool)) { }
```

Comment:
Is this just an automated change from the genapi tool?

Reply:
yes, it is. I was thinking that it would be nice to update ref assemblies always on tool updates so this is weeded out. Or I can cherry pick just the relevant changes.